code/game/objects/items/devices/radio/electropack.dm

```diff
@@ -27,13 +27,14 @@
 	SSradio.remove_object(src, frequency)
 	return ..()
 
+//ATTACK HAND IGNORING PARENT RETURN VALUE

```

Comment:
Isnt there another method we could do here?

Reply:
What do you mean? For the comment or the proc?

Reply:
That was the proc.